GoToProject/src/containers/SignUp/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {RecordHeader} from '../../components/RecordHeader';
+import { Layout } from '../../components/styled';
+import { CustomInput } from '../../components/customInput';
+import { ButtonCustom } from '../../components/ButtonCustom';
+
+export const SignUp = () => {
+
+  return (   
+        <Layout>
+          <RecordHeader title="Sign Up" />
+          <CustomInput label="Full Name" />
+          <CustomInput label="Email" />
+          <CustomInput label="Password" />
```

Comment:
could you refactor this?

Reply:
any suggestion? I already reduced it from writting 3 times the input box and label code (custom input component includes the input and the label), to just write it once and call each time needed, for this screen we need 3 inputs.

Reply:
you can use a map() or section/flat list

Reply:
<CustomInput label="{input name}" action=(action) />

const login = [
  {
    title: 'Full name',
   action: () => {action()}
  },
  {
    title: 'email',
  },
  {
    title: 'password',
  },
];

Reply:
Ya actualice el archivo, y ahora utilizo map() para mostrar el componente que se repetia.

Reply:
I have updated the file and now I use map() to display the component without repeating the code.